### PR TITLE
Add style description

### PR DESCRIPTION
**Pull request by jimcat8**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

File: docs/books/bash_programming/01-Shell-overview.md [hunk 1]
```diff
@@ -35,11 +35,133 @@ Shell classification:
 * Bourne Shell - This family includes but is not limited to:
     * sh (Bourne Shell, /usr/bin/sh). It was developed at Bell LABS in 1977 by Stephen Bourne and used on V7 UNIX
     * ksh (Korn Shell, /usr/bin/ksh)
-    * Bash (GNU Bourne-Again Shell, /bin/bash) - Born in 1987, it is the product of the GNU Project. Most GNU/Linux operating systems use bash as their default shell
-    * psh (POSIX Shell)
-    * zsh (Z-shell)
+    * Bash (GNU Bourne-Again Shell, /bin/bash) - Born in 1987, it is the product of the GNU Project. Most GNU/Linux operating systems use bash as their default shell. For more detailed information, please [read here](https://www.gnu.org/software/bash/manual/bash.html).
```
Comment (SergeCroise) [inline docs/books/bash_programming/01-Shell-overview.md:38]:
```suggestion
    * Bash (GNU Bourne-Again Shell, /bin/bash) - Born in 1987, it is the product of the GNU Project. Most GNU/Linux operating systems use bash as their default shell. For more detailed information, please [read this manual](https://www.gnu.org/software/bash/manual/bash.html).
```

**Review by SergeCroise — COMMENTED** [review]
hi @jimcat8 ,
Feel free to overwrite or reverse my modifications.

File: docs/books/bash_programming/01-Shell-overview.md [hunk 2]
```diff
@@ -35,11 +35,133 @@ Shell classification:
...
+    * zsh (Z-shell) - For more detailed information, please [read here](https://zsh.sourceforge.io/).
 * C Shell - This family includes but is not limited to:
     * csh
-    * tcsh
+    * tcsh - For more detailed information, please [visit this website](https://www.tcsh.org/)
 * Power Shell
 
+## Fence Code Block style description
+
+* Translators should translate the annotations on the command.
```
Comment (wsoyinka) [inline docs/books/bash_programming/01-Shell-overview.md:48]: @jimcat8  - Who are notes like these ^^^ meant for ? 

I could be mistaken but -  if they are meant for translators or editors or other contributors then they don't belong directly in this book/guide/doc.  
And if the notes are meant as a form of reminder to yourself to do something later on - please convert this to work into a draft.

File: docs/books/bash_programming/01-Shell-overview.md [hunk 3]
```diff
@@ -35,11 +35,133 @@ Shell classification:
...
+    17  ├── Rocky-RT.repo
+    18  └── Rocky-Sources.repo
+    19
+    20  0 directories, 17 files
+  ```
+
+   * Epel repository - Lines 2 to 5
+   * Rocky repository - Lines 6 to 18
+  
+* Sometimes, in order to better illustrate the author's intended meaning, adjustments can be made to the output content, such as:
```
Comment (wsoyinka) [inline docs/books/bash_programming/01-Shell-overview.md:143]: @jimcat8  - Who are notes like these ^^^ meant for ? 

I could be mistaken but -  if they are meant for translators or editors or other contributors then they don't belong directly in this book/guide/doc.  
And if the notes are meant as a form of reminder to yourself to do something later on - please convert this to work into a draft.

File: docs/books/bash_programming/01-Shell-overview.md [hunk 4]
```diff
@@ -35,11 +35,133 @@ Shell classification:
...
+  bash > cat /etc/services
+  ...
+  discard         9/tcp           sink null
+  discard         9/udp           sink null
+  systat          11/tcp          users
+  systat          11/udp          users
+  ...
+  ```
+
+* If you want to explain a part of the output content, please add line numbers to the output using pipeline symbols.
```
Comment (wsoyinka) [inline docs/books/bash_programming/01-Shell-overview.md:114]: @jimcat8  - Who are notes like these ^^^ meant for ? 

I could be mistaken but -  if they are meant for translators or editors or other contributors then they don't belong directly in this book/guide/doc.  
And if the notes are meant as a form of reminder to yourself to do something later on - please convert this to work into a draft.

File: docs/books/bash_programming/01-Shell-overview.md [hunk 5]
```diff
@@ -35,11 +35,133 @@ Shell classification:
...
+  devtmpfs       devtmpfs  1.8G     0  1.8G   0% /dev
+  tmpfs          tmpfs     1.8G     0  1.8G   0% /dev/shm
+  tmpfs          tmpfs     1.8G  8.9M  1.8G   1% /run
+  tmpfs          tmpfs     1.8G     0  1.8G   0% /sys/fs/cgroup
+  /dev/nvme0n1p2 ext4       46G  4.0G   40G  10% /
+  /dev/nvme0n1p1 xfs      1014M  248M  767M  25% /boot
+  tmpfs          tmpfs     363M     0  363M   0% /run/user/0
+  ```
+
+* If the output content of the command is too much, the author will use "..." to indicate it.
```
Comment (wsoyinka) [inline docs/books/bash_programming/01-Shell-overview.md:102]: @jimcat8  - Who are notes like these ^^^ meant for ? 

I could be mistaken but -  if they are meant for translators or editors or other contributors then they don't belong directly in this book/guide/doc.  
And if the notes are meant as a form of reminder to yourself to do something later on - please convert this to work into a draft.

File: docs/books/bash_programming/01-Shell-overview.md [hunk 6]
```diff
@@ -35,11 +35,133 @@ Shell classification:
...
+  
+  ```bash
+  bash (root /root/)> ls -l
+
+  bash (root)>
+
+  bash (/tmp/)>
+  ```
+
+* Use arrows to illustrate the meaning of command output.
```
Comment (wsoyinka) [inline docs/books/bash_programming/01-Shell-overview.md:74]: @jimcat8  - Who are notes like these ^^^ meant for ? 

I could be mistaken but -  if they are meant for translators or editors or other contributors then they don't belong directly in this book/guide/doc.  
And if the notes are meant as a form of reminder to yourself to do something later on - please convert this to work into a draft.

**Review by wsoyinka — CHANGES_REQUESTED** [review]

**Pull request closed without merging**